jdat_notebooks/asdf_example/asdf_example.ipynb

```diff
@@ -4,7 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Example of Converting a FITS File to ASDF\n",
+    "# ASDF Example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example of Converting a FITS File to ASDF\n",
```

Comment:
It looks like the title has been separated into two cells here. Is that intentional?